Dockerfile

```diff
@@ -6,6 +6,12 @@ WORKDIR /usr/src/epub-press
 COPY package.json package-lock.json ./
 RUN npm install --production
 
+# Automatically downloads and installs Calibre. To use Kindlegen instead, set ARG converter=kindlegen.
+ARG converter=calibre
+ENV CONVERSION_BACKEND=$converter
+RUN if [ "$converter" = "calibre" ] ; then apt-get update && apt-get install -y libgl1-mesa-glx && wget -nv -O- https://download.calibre-ebook.com/linux-installer.sh && sh /dev/stdin install_dir=calibre-bin/ isolated=y ; fi
```

Comment:
Maybe this should be:

```
if [ $CONVERSION_BACKEND = "calibre" ] ; then ...
```

Running this locally in the container seems to be a no-op... maybe because the arg is only set at build time, but the "" around the variable still seem suspect...

Reply:
Nvm... looking at the build it seems like it ran...

https://github.com/haroldtreen/epub-press/runs/4493960629?check_suite_focus=true

Reply:
The only reason I went for the ARG of `converter` and the ENV of `CONVERSION_BACKEND` is because I wasn't sure what would happen if both a build arg and an environment variable had the same name.

Reply:
Disregard! This wasn't the issue and confirmed it's working 👌.

![image](https://user-images.githubusercontent.com/1745854/145687620-f6821887-015e-49c3-bfe5-f98a716503ec.png)